module/GameMotion/PartsofGameMotion.cpp

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file PartsofGameMotion.cpp
```

Comment:
こういう時はofも大文字にして大丈夫だよ
```suggestion
 * @file PartsOfGameMotion.cpp
```